runtime/modules/clipboard.lua

```diff
@@ -0,0 +1,42 @@
+local code = require("vscode-neovim")
+
+local last_item = nil
+
+local function paste()
+  local curr_text = code.call("clipboard_read"):gsub("\r\n", "\n")
+  local curr_item = { vim.split(curr_text, "\n"), "v" }
+
+  if not last_item then
+    return curr_item
+  end
+
+  if not vim.deep_equal(last_item[1], curr_item[1]) then
+    return curr_item
+  end
+
+  return last_item
+end
+
+local function copy(lines, regtype)
+  last_item = { lines, regtype }
+  local text = table.concat(lines, "\n")
+  code.call("clipboard_write", { args = { text } })
+end
+
+vim.g.vscode_clipboard = {
```

Comment:
Don't we have a "vscode" Lua moduel? Why define a new `g:` variable? Instead we can document something like:

    vim.g.clipboard = require('vscode').clipboard

Reply:
Are we not deprecating the vimscript interface? It is confusing to have both.

> it is not convenient to set it in Vimscript if it is exported as a Lua field.

Setting it in vimscript is: 

    let g:clipboard = v:lua.require('vscode-neovim').clipboard

, which we can document. Does it need to be more convenient than that?

`require('vscode-neovim')` is awkward to type, but that is a general issue unrelated to this specific case. (I don't remember why `'vscode-neovim'` was chosen as the module name instead of 'vscode')

Reply:
Someone complained that it collided with their existing config and/or "vscode" theme. In hindsight, we should have told them tough luck.

Reply:
> was chosen as the module name instead of `'vscode'`)

This is what I've always wanted and was discussed while working on 1.0

Reply:
We could potentially make an alias to avoid annoyance.